# Generated by Django 2.1.1 on 2018-10-15 16:44

from django.db import migrations, models
import user.models


class Migration(migrations.Migration):

    dependencies = [
        ('user', '0007_auto_20181014_2036'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='guild_points',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[user.models.MyValidator()], verbose_name='username'),
        ),
    ]
